File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/assistant_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class AssistantResponse:
    """助手响应"""
    id: str = None
    object: str = None
    model: str = None
    name: str = None
    description: str = None
    instructions: str = None
    tools: List[Any] = None
    metadata: Dict[str, str] = None
    temperature: float = None
    created_at: int = None
    tool_resources: Dict[str, Any] = None
    response_format: Any = None
    top_p: float = None

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/AssistantResponse.kt
package com.sdkwork.ai

data class AssistantResponse(
    val id: String? = null,
    val object: String? = null,
    val model: String? = null,
    val name: String? = null,
    val description: String? = null,
    val instructions: String? = null,
    val tools: List<Any>? = null,
    val metadata: Map<String, String>? = null,
    val temperature: Double? = null,
    val createdAt: Int? = null,
    val toolResources: Map<String, Any>? = null,
    val responseFormat: Any? = null,
    val topP: Double? = null
)
